pkg/nsx/services/vpc/vpc.go

```diff
@@ -30,14 +31,18 @@ const (
 	GroupKey                   = "/orgs/%s/projects/%s/vpcs/%s/groups/%s"
 	SecurityPolicyKey          = "/orgs/%s/projects/%s/vpcs/%s/security-policies/%s"
 	RuleKey                    = "/orgs/%s/projects/%s/vpcs/%s/security-policies/%s/rules/%s"
+	albEndpointPath            = "policy/api/v1/infra/sites/default/enforcement-points/alb-endpoint"
+	NSX_LB                     = "nsx-lb"
+	AVI                        = "avi"
```

Comment:
`NSX_LB` - > `LBProviderNSX`
`AVI` -> `LBPrivoderAVI`
or some other names like that. Otherwise, `NSX_LB` doesn't match the naming style, and these 2 var name doesn't include their purpose.

Reply:
fixed